ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -11,9 +11,9 @@ if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
   exit 1
 fi
 
+echo "Installing from release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
 export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
 export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
-export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${RELEASE_IMAGE_LATEST}
```

Comment:
And also adjust the `RELEASE_IMAGE_LATEST is an empty string, exiting` guard above^^ to use `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`.